app/_hub/kong-inc/ai-rate-limiting-advanced/overview/_index.md

```diff
@@ -94,10 +94,13 @@ The plugin supports three strategies to calculate the token:
 
 | Strategy    | Description |
 | --------- | ---- |
-| `total_tokens`   | Represents the total number of tokens, including both the prompt and the generated completion, in the LLM's input sequence. 
-| `prompt_tokens` | Represents the tokens provided by the user as input to the LLM, typically defining the context or task.
-| `completion_tokens`   | Represents the tokens generated by the LLM in response to the prompt, representing the completed output or continuation of the task.
+| `total_tokens`   | Represents the total number of tokens, including both the prompt and the generated completion, in the LLM's input sequence. |
+| `prompt_tokens` | Represents the tokens provided by the user as input to the LLM, typically defining the context or task. |
+| `completion_tokens`   | Represents the tokens generated by the LLM in response to the prompt, representing the completed output or continuation of the task. |
 
+{% if_version gte: 3.8.x %}
+| `cost`   | Represents the financial or computational cost incurred based on the tokens used by the LLM during the request. Using this strategy can help you limit API usage based on the actual costs of processing the request, ensuring that expensive requests (in terms of token usage) are managed more carefully. This cost is calculated by taking the sum of multiplying the number of prompt tokens by the cost per prompt token (input cost) and by multiplying the number of completion tokens by the cost per completion token (output cost): `cost = prompt_tokens * input_cost + completion_tokens * output_cost`. The `input_cost` and `output_cost` fields must be defined in the AI plugins for this to work. |
```

Comment:
> The `input_cost` and `output_cost` fields must be defined in the AI plugins for this to work.

@AntoineJac Are those fields in this AI Rate Limiting Advanced plugin, or are these in a different AI plugin(s)?

Reply:
@cloudjumpercat  it is a new field pushed in 3.8 for the AI Rate Limiting Advanced plugin